### PR TITLE
Convert add feature diagrams implemented using PlantUML 

### LINKED ISSUE #221: Add feature diagrams in DG are not updated and implemented via PlantUML

### PR DESCRIPTION
**Pull request by OVReader**
Fix #221

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ The first section is facilitated by the following classes:
 The following is a simple class diagram of the three classes:
 <p align="center">
 
-![](diagrams/ParseAddRelatedClassesDiagram.jpg)
+![](diagrams/ParseAddRelatedClassesDiagram.png)
```
Comment (wilsonngja) [inline docs/DeveloperGuide.md:148]: The class diagram seems to be different from what was taught in class. Perhaps there can be different setting to it? 

If there isn't can we consider not using PlantUML for class diagram?

File: docs/diagrams/AddClientSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+@startuml
+hide footbox
+
+participant "command:CommandAddClient" as CommandAddClient #lightgreen
+participant "clientList:ClientList" as ClientList #lightblue
+participant ":Client" as Client #yellow
+participant "ui:Ui" as Ui #lightgrey
+participant "storage:Storage" as Storage #orange
```
Comment (wilsonngja) [inline docs/diagrams/AddClientSequenceDiagram.puml]: Would it be good for the team to all stay to a specified colour scheme?

**Review by wilsonngja — COMMENTED** [review]
Great effort in the addition of porting over to PlantUML. With regards to the class diagram, would it be possible to look it up to see if we can change the settings to the one that fits the one that is taught in class?

File: docs/diagrams/AddClientSequenceDiagram.puml (hunk 2 above)
Comment (ngdeqi) [reply docs/diagrams/AddClientSequenceDiagram.puml]: > Would it be good for the team to all stay to a specified colour scheme?

My style.puml ;-;

**Review by ngdeqi — COMMENTED** [review]
Class Diagram has a bit too much info. Otherwise, LGTM

**Review by FeliciaBeatrice — APPROVED** [review]
LGTM!

**Pull request merged**